doc/source/index.rst

```diff
@@ -116,30 +116,29 @@ Ray provides Python, Java, and *EXPERIMENTAL* C++ API. And Ray uses Tasks (funct
 
     | The C++ Ray API is currently experimental with limited support. You can track its development `here <https://github.com/ray-project/ray/milestone/17>`__ and report issues on GitHub.
     | Run the following commands to get started:
-    | - Build ray from source with *bazel* as shown `here <https://docs.ray.io/en/master/development.html#building-ray-full>`__.
-    | - Modify and build `cpp/example/example.cc`.
+
+    | - Install ray with C++ API support and generate bazel project template by ray script.
 
     .. code-block:: shell
 
-      pip install -e python --verbose
-      cd cpp/example/
-      bazel build //:example
+      pip install ray[cpp]
+      mkdir ray-template && ray cpp --generate-bazel-project-template-to ray-template
 
-    | Option 1: run the example directly with a dynamic library path. It will start a Ray cluster automatically.
+    | - Option 1: run the example directly. It will start a Ray cluster automatically.
```

Comment:
The project template comes with a simple example application. You can try this example out in 2 ways:
1. Run the example application directly, which will start a Ray cluster locally.
2. Connect the example application to an existing Ray cluster by specifying the `RAY_ADDRESS` env var.